kf_lib_data_ingest/common/concept_schema.py

```diff
@@ -7,9 +7,9 @@
 class FileMixin(object):
     ETAG = None
     SIZE = None
-    DATA_TYPE = None
+    HASH = None
```

Comment:
Yea I think this will have to be a dict bc a file can have more than one hash and we have to know which value goes with which type. Right now all we can do is call it `HASHES` and then leave it to the developer of the extract config to map a dict to it like in [this example](https://github.com/kids-first/kf-ingest-packages/blob/26f8c1981278eb501196f50a840cbe03eb4155ac/kf_ingest_packages/common/extract_configs/s3_object_info.py#L145) 

I think we definitely need to sit down and discuss design of both concept schema and target API config schema bc neither one of them account for types